site/docs/guide/deployment-types.md

```diff
@@ -30,15 +30,15 @@ _Imagine you're getting yourself a scoop of ice cream in your trusted ice cream
 You walk up to the employee and ask for your favorite type of ice cream.
 Unfortunately, he lets you know that it is out of stock._
 
-_The next day, you're craving that delicious ice cream again, so you go back to the same place and ask for the same ice cream.
+_The next day, you're craving that delicious ice cream again, so you return to the same place and ask for the same ice cream.
 Good news!
-They restocked over night so you can enjoy your salted caramel ice cream today!
+They are restocked overnight, so you can enjoy your salted caramel ice cream today!
 Yummy._
 
 **Polling** means that grammY proactively sends a request to Telegram, asking for new updates (think: messages).
-If no messages are there, Telegram will return an empty list, indicating that no new messages were sent to your bot since the last time you asked.
+Telegram will return an empty list if no messages are there, indicating that no new messages have been sent to your bot since the last time you asked.
```

Comment:
Please revert the former change. It's useful to stick with the convention of listing the condition first when examining all cases.